RCNN_tf.py

```diff
@@ -186,7 +197,7 @@ def accuracy(
     test_data_set_ = tf.data.Dataset.from_tensor_slices((
         input_placeholder_,
         output_placeholder_
-    )).batch(batch_size=batch_size_)
+    )).batch(batch_size=batch_size_).shuffle(buffer_size=buffer_size_)
```

Comment:
I believe this ordering needs to be changed to 
1. `shuffle`
2. `repeat`
3. `batch`
check if this is correct, but i think the way it is now it will shuffle the batches, but the order in the batch will be the same as it's read.

Reply:
Ah yea right. Changed!